pulp_2to3_migration/app/tasks/migrate.py

```diff
@@ -107,6 +112,14 @@ def get_repo_types(plan):
 
     task_group = TaskGroup(description="Migration Sub-tasks")
     task_group.save()
+    GroupProgressReport(
+        message="Repo migration",
+        code="create.repo_version",
+        task_group=task_group).save()
+    GroupProgressReport(
+        message="Repo migration",
+        code="create.distribution",
+        task_group=task_group).save()
```

Comment:
Should messages differ?

Reply:
I probably have not articulated it in the best way - but the intention was to highlight that a repo is being migrated - and repo creation and distribution creation is part of its migration.
But now i see how it can be confusing. I will then rename to 'Repo Version Creation" and "Distribution Creation"